app/services/iaa_reporting_helper.rb

```diff
@@ -74,4 +74,8 @@ def partner_accounts
         end
       end.compact
   end
+
+  def key(gtc_number, order_number)
```

Comment:
I would make these keyword args so its harder to get them in the wrong order on accident
```suggestion
  def key(gtc_number:, order_number:)
```